src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -22,54 +22,61 @@ clusters with overlapping CIDRs to connect together.
 
 To support overlapping CIDRs in connected clusters, Submariner has a component called Global Private Network, Globalnet (`globalnet`). This
 Globalnet is a virtual network specifically to support Submariner's multi-cluster solution with a global CIDR. Each cluster is given a
-subnet from this virtual Global Private Network, configured as new cluster parameter `GlobalCIDR` (e.g. 169.254.0.0/16) which is
+subnet from this virtual Global Private Network, configured as new cluster parameter `GlobalCIDR` (e.g. 242.0.0.0/8) which is
 configurable at time of deployment. User can also manually specify GlobalCIDR for each cluster that is joined to the Broker using the flag
 ```globalnet-cidr``` passed to ```subctl join``` command. If Globalnet is not enabled in the Broker or if a GlobalCIDR is preconfigured in
 the cluster, the supplied globalnet-cidr will be ignored.
 
-Once configured, each exported Service and Pod that requires cross-cluster access is allocated an IP, named `globalIp`,
-from this `GlobalCIDR` that is annotated on the Pod/Service object.
-This globalIp is used for all cross-cluster communication to and from a Pod and the globalIp of a
-remote Service. Routing and iptable rules are configured to use the globalIp for ingress and egress. All address translations occur on the
-Gateway node.
+### Cluster-scope global egress IPs
 
-![Figure 1 - Proposed solution](/images/globalnet/overlappingcidr-solution.png)
+By default every cluster is assigned a number of global IPs which is used as egress IPs for any cross cluster communication.
+Multiple IPs are supported to avoid ephemeral port exhaustion issues. Default is `8`.
+Applications running on the host network that access remote clusters also use the cluster-level global egress IPs.
 
-{{% notice info %}}
+### Namespace-scope global egress IPs
 
-Unlike vanilla Submariner, where Pod to Pod connectivity is also supported, Globalnet only supports Pod to remote Service connectivity using
-globalIps.
+A user can assign a configurable number of global IPs per namespace by creating a `GlobalEgressIP` resource. These IPs are also allocated
+from the global CIDR and are used as egress IPs for all or selected pods in the namespace and take precedence over the cluster-level global IPs.
 
-{{% /notice %}}
+User can also configure `GlobalEgressIP`s for a specific set of pods in a namespace by using a `PodSelector` when creating `GlobalEgressIP`.
+These `GlobalEgressIP`s will take precedence over any global IPs allocated at namespace or cluster level.
```

Comment:
```suggestion
```

Reply:
@tpantelis Why this change? We need to specify that pod-set specific egressIPs will take precedence over Namespace.

Reply:
Right - I changed the wording above to include this point.